examples/solids/index.rst

```diff
@@ -316,27 +316,21 @@ Carrying through the differentiation :math:numref:`strain-energy-grad` for the m
 Weak form
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is crucial to distinguish between the current and reference element in the Total Lagrangian Finite Strain regime. Therefore, we switch to the indicial notation:
+It is crucial to distinguish between the current and reference element in the Total Lagrangian Finite Strain regime.
 
 .. math::
 
-    \int_{B}{\boldsymbol{v} \cdot \left(\nabla_X \cdot \boldsymbol{P} + \rho \boldsymbol{g}\right)} JdV = \boldsymbol{0}
-
-and in indicial notation we have,
-
-.. math::
-
-   \int_{B_0}{v_i \left(\dfrac{\partial{P_{iI}}}{\partial{X_I}} + \rho_0 g_i \right)} dV = 0
+    \int_{\Omega}{\boldsymbol{v} \cdot \left(\nabla_X \cdot \boldsymbol{P} + J \rho \boldsymbol{g}\right)} dV = \boldsymbol{0}
```

Comment:
```suggestion
    \int_{\Omega}{\boldsymbol{v} \cdot \left(\nabla_X \cdot \boldsymbol{P} + \rho_0 \boldsymbol{g}\right)} dV = \boldsymbol{0}
```
We don't know `\rho` in the current configuration (because it depends on the solution), only `\rho_0`.

Reply:
It warrants a comment that the body force depends on the current configuration and needs to be mapped back to the initial configuration. This is another form of nonlinearity.